src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -230,6 +240,39 @@ class ProductaanvraagService @Inject constructor(
         return initiatorBetrokkene
     }
 
+    private fun Betrokkene.shouldBeSkipped(
+        zaak: Zaak,
+        genericRole: Boolean,
+        brpEnabled: Boolean,
+        kvkEnabled: Boolean
+    ): String? {
+        val rolTypeDescription = this.roltypeOmschrijving ?: this.rolOmschrijvingGeneriek.toString()
+        val prefix = if (genericRole) "generic " else ""
+
+        return this.performAction(
+            onNatuurlijkPersoonIdentity = {
+                if (!brpEnabled) {
+                    "Betrokkene with ${prefix}roletype description `$rolTypeDescription` has BRP-based identity, but BRP " +
+                        "is not enabled for zaak type ${zaak.zaaktype}. No betrokkene role created for zaak ${zaak.identificatie}"
+                } else {
+                    null
+                }
+            },
+            onKvkIdentity = { _, _ ->
+                if (!kvkEnabled) {
+                    "Betrokkene with ${prefix}roletype description `$rolTypeDescription` has KvK-based identity, but KvK " +
```

Comment:
```suggestion
                    "Betrokkene with ${prefix}roletype description `$rolTypeDescription` has KVK-based identity, but KVK " +
```

We use `KVK` everywhere else.